src/library_webgl.js

```diff
@@ -2699,19 +2684,30 @@ var LibraryGL = {
     return GLctx.getAttribLocation(GL.programs[program], UTF8ToString(name));
   },
 
-  glGetActiveAttrib__sig: 'viiiiiii',
-  glGetActiveAttrib: function(program, index, bufSize, length, size, type, name) {
+  _glGetActiveAttribOrUniform: function(funcName, program, index, bufSize, length, size, type, name) {
```

Comment:
Use $ prefix is this is JS-only?   Or maybe hang it of the already-internal GL object?

Reply:
In the olden days functions that did not have any prefix were understood to be public for Emscripten JS library users, (e.g. `UTF8ToString`). Not sure if that scheme changed at some point? The intent of `_glGetActiveAttribOrUniform` is not not be called by users, hence the underscore.

(Also hanging items on the `GL` object is a delicate balance with Closure, it often increases code size)